src/server/saved_objects/service/lib/repository.test.js

```diff
@@ -1649,12 +1681,69 @@ describe('SavedObjectsRepository', () => {
       onBeforeWrite.throws(es401);
 
       try {
-        await savedObjectsRepository.delete('type', 'id');
+        await savedObjectsRepository.delete('foo', 'id');
       } catch (error) {
         sinon.assert.calledOnce(onBeforeWrite);
         expect(error).toBe(es401);
         expect(errors.isNotAuthorizedError(error)).toBe(true);
       }
     });
   });
+
+  describe('hidden types', () => {
```

Comment:
Since the test code itself is determining what a "hidden" type is, it might be more accurate to say that you're testing unknown or unsupported types, rather than hidden types. The repository doesn't have knowledge of what is or isn't a hidden type, it's just given a list of types it needs to support.

Reply:
This is true, and makes a lot of sense. I will change the name of the suite.